app/controllers/application_controller.rb

```diff
@@ -134,6 +134,20 @@ def require_moderator
     end
   end
 
+  ##
+  # require that the user is a administrator, or fill out a helpful error message
+  # and return them to the user/login page.
+  def require_administrator
+    if current_user
+      unless current_user.administrator?
+        flash[:error] = t("user.filter.not_an_administrator")
```

Comment:
That needs to be `users.filter.not_an_administrator` as that key doesn't exist - looks like you chose to copy the one that is broken since @gravitystorm renamed `user` to `users` last week ;-)

The `user_role` version can also be removed from `config/locales/en.yml` as it will no longer be used once this is merged.

Reply:
Incidentally the fact we hadn't picked that up means that the "need administrator" path in the diary entry controller is not being tested, so if you feel like adding a test for that it would be great ;-)